pipeline/email_notifier.py

```diff
@@ -14,12 +14,12 @@ def get_ses_client() -> boto3.client:
     """Returns the boto3 SES client to send emails with."""
 
     aws_access_key = ENV.get("AWS_ACCESS_KEY_ID")
-    aws_secret_access_key = ENV.get("AWS_ACCESS_SECRET_KEY")
+    aws_secret_access_key = ENV.get("AWS_SECRET_ACCESS_KEY")
```

Comment:
Thank you so much for spotting this error, can't believe I added this in :')

Reply:
Thanks!